plugins/tekton/README.md

```diff
@@ -47,9 +47,15 @@ This plugin will help with visualizing the workloads like Deployment, Replicaset
      backstage.io/kubernetes-id: <BACKSTAGE_ENTITY_NAME>
      backstage.io/kubernetes-namespace: <RESOURCE_NS>
    ```
-6. A custom label selector can also be added which will then be used by Backstage to find the resources. The label selector takes precedence over the id annotation.
+6. To view the latest PipelineRun visualization in the CI/CD tab of the application, add the below annotation.
+   ```yaml
+   annotations:
+     ...
+     tekton.dev/build-namespace: <RESOURCE_NS>
```

Comment:
Out of curiosity, why `tekton.dev/` is used here ? usually `tekton.dev` and `*.tekton.dev` labels are automatically added by Tekton and it will be used internally by tekton. IMO we shouldn't be using this prefix to connect backstage to a namespace. why not `backstage.io/build-namespace` ?

Reply:
I tried to follow this document here https://backstage.io/docs/features/software-catalog/well-known-annotations/#jenkinsiojob-full-name , I can update this if you still feel we should go with "backstage.io/build-namespace"

Reply:
can we use `janus.idp/*` prefix for the plugins that we maintain?

Reply:
yeah it make sense to have custom and not conflict with others and namespace doesn't make sense here as there could be resources across namespaces so might confuse user and we are not using the namespace either way

How about

`janus.idp/tekton-enabled : true` // by default this will be enabled